### PR TITLE
[CAM-11114] Variables section for DP boxes (Cloud Academy)

### PR DESCRIPTION
**Pull request by SergioQCL**
@gsanchezu @sunew please take a look

### CODE REVIEW

File: Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
 {{{ "title": "Deployment Policy Boxes",
-"date": "09-19-2018",
-"author": "",
-"date": "12-28-2018",
-"author": "Julio Castanar",
+"date": "05-20-2019",
+"author": "Julio Castanar & Sergio Quintana",
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md]:
```suggestion
"author": "Julio Castanar and Sergio Quintana",
```

File: Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md [hunk 2]
```diff
@@ -89,6 +87,26 @@ Give access to other individuals or team workspaces by clicking on the pencil of
 
 For more information about sharing, see [sharing boxes, instances and providers](../Core Concepts/workspaces-and-collaboration.md#sharing-boxes-instances-and-providers).
 
+### Deployment policy variables
+
+Clicking the *Code* section of a Deployment Policy box gives the user the possibility of editing the already defined Policy, as well as defining some variables to parametrize that box.
+
+![Available variable types for Deployment Policy boxes](../../images/cloud-application-manager/deploymentpolicyboxes-variables.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md]: There is a folder for deployment-policy images, please move this one there

File: Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md [hunk 3]
```diff
@@ -89,6 +87,26 @@ Give access to other individuals or team workspaces by clicking on the pencil of
...
+
+Almost all kinds of variables which apply in this case are already explained in detail [here](parameterizing-boxes-with-variables.md), so a general overview will be given:
+
+* Box variable: holds a reference of the configuration of another box in your workspace. This is how multi-tier applications can be achieved. Child boxes inside will deploy on the same virtual infrastructure as the parent box.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md]: Please, use bold for the variable types in all:
```suggestion
* **Box variable**: holds a reference of the configuration of another box in your workspace. This is how multi-tier applications can be achieved. Child boxes inside will deploy on the same virtual infrastructure as the parent box.
```

File: Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md (hunk 2 above)
Comment (SergioQCL) [reply Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md]: All images of this article are stored in cloud-application-manager folder. Should I move all of them to deployment-policy?

Comment (SergioQCL) [reply Cloud Application Manager/Automating Deployments/deploymentpolicy-box.md]: Already moved 👍

**Pull request merged**